guides/source/developers/customizations/customizing-assets.html.md

```diff
@@ -0,0 +1,183 @@
+`vendor/assets` folder.
+
+A typical `vendor/assets` directory for a Solidus application will look like:
+
+    vendor
+    |-- assets
+        |-- images
+        |   |-- spree
+        |       |-- frontend
+        |       |-- backend
+        |-- javascripts
+        |   |-- spree
+        |       |-- frontend
+        |       |   |-- all.js
+        |       |-- backend
+        |           |-- all.js
+        |-- stylesheets
+        |   |-- spree
+        |       |-- frontend
+        |       |   |-- all.css
+        |       |-- backend
+        |           |-- all.css
+
+
+As you can see, Solidus generates four top-level manifests (`all.css` &
+`all.js`, see above) that require all the Solidus and site-specific
+stylesheets/JavaScript files.
+
+For example, here's the content of the
+`vendor/assets/stylesheets/frontend/all.css` manifest:
+
+```css
+/*
+ *= require spree/frontend
+ *= require_self
+ *= require_tree .
+*/
+```
+
+As you can see, it requires `spree/frontend` stylesheets from
+the solidus gem and all the files under the current path, recursively.
+
+The `frontend/all.css` manifest is loaded into the host application via a
+standard [stylesheet_link_tag][stylesheet-link-frontend] in the frontend
+erb main layout.
+
+This structure allows you to load all the Solidus assets and also gives you
+the ability to fully customize them. We'll see how in the next few paragraphs.
+
+[stylesheet-link-frontend]: https://github.com/solidusio/solidus/blob/9ae2ed16bad7e29ea422fab1403118a3a0e66266/frontend/app/views/spree/shared/_head.html.erb#L8
+
+## Overriding Solidus core assets
+
+Overriding or replacing any of Solidus's internal assets is very easy.
+It's recommended to attempt to replace as little as possible in a given
+JavaScript or stylesheet file to help ease future upgrade work required.
+
+The methods listed below works for both frontend and backend, and it
+also applies to extensions that provide assets.
+
+### Overriding individual CSS styles
+
+For example, let's suppose that you want to change how the footer looks,
+which is defined by the following rule-set:
+
+```css
+/* solidus/frontend -> app/assets/stylesheets/spree/frontend/screen.css */
+
+div#footer {
+ clear: both;
+}
+```
+
+You can create a new stylesheet inside
+`vendor/assets/stylesheets/spree/frontend/` and include the
+following CSS:
+
+```css
+/* your_app -> vendor/assets/stylesheets/spree/frontend/foo.css */
+
+div#footer {
+ clear: none;
+ border: 1px solid red;
+}
+```
+
+
+The `frontend/all.css` manifest will automatically include `foo.css`, due to
+this directive that includes all the files in the current directory:
+
+```css
+/*
+ *= require_tree .
+*/
+```
+
+Now both rule-sets are loaded and standard CSS specificity rules are applied
+to determine the final result for that element. In this case, since the
+`require_tree` directive is defined after `require spree/frontend`, our
+new rule-set will be lower in the final stylesheet and will take precedence.
```

Comment:
not sure about `will be lower`: maybe something like `will be loaded later` works better?

Reply:
You are right, it's not super clear, I'm going with `will be evaluated later`, does it work?